frontend/public/components/app.jsx

```diff
@@ -61,6 +61,7 @@ const NOTIFICATION_DRAWER_BREAKPOINT = 1800;
 // Edge lacks URLSearchParams
 import 'url-search-params-polyfill';
 import { graphQLReady } from '../graphql/client';
+import '@patternfly/patternfly/patternfly-theme-dark.scss'; // load dark theme here as MiniCssExtractPlugin ignores load order of sass and dark theme must load after all other css
```

Comment:
Can this go on line 55 so it's with the other (s)css?

Reply:
Yes, if line 57 can move to line 52, above the css imports

Reply:
Sure.  The goal is to keep things grouped.

Reply:
@rhamilto updated